src/fastcs/transports/epics/ca/ioc.py

```
@@ -218,18 +218,25 @@ async def async_record_set(value: DType_T):
+    alias,
+    attribute,
+):
     pv = f"{pv_prefix}:{pv_name}"
 
     async def on_update(value):
         logger.info("PV put: {pv} = {value}", pv=pv, value=repr(value))
-
-        await attribute.put(cast_from_epics_type(attribute.datatype, value))
+        try:
+            await attribute.put(cast_from_epics_type(attribute.datatype, value))
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)